sdk/examples/wallet/12_increase_native_token_supply.rs

```diff
@@ -38,14 +38,21 @@ async fn main() -> Result<()> {
 
     // Mint some more native tokens
     let mint_amount = U256::from(10);
-    let mint_transaction = account
+    let transaction = account
         .increase_native_token_supply(token_id, mint_amount, None, None)
         .await?;
+    println!("Transaction sent: {}", &transaction.transaction.transaction_id);
```

Comment:
Why do you print by reference btw?

Reply:
just a habit, so the value doesnt move and you can always copy paste

Reply:
`println!` automatically takes reference so it never moves anything, hence why it's a bit unusual to see a reference in the call

Reply:
huh i swear i had this before, but reading up on it makes no sense